### PR TITLE
Release 1.6.0

### PR DESCRIPTION
**Pull request by jakirkham**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,12 +23,16 @@ requirements:
   run:
     - python
     - absl-py >=0.1.6
+    - astor >=0.6.0
     - enum34 >=1.1.6  # [py<34]
+    - gast >=0.2.0
```
Comment (jakirkham) [inline recipe/meta.yaml]: Currently `gast` is not packaged. Have raised a request for its addition in issue ( https://github.com/conda-forge/staged-recipes/issues/5704 ).

File: recipe/meta.yaml [hunk 2]
```diff
@@ -23,12 +23,16 @@ requirements:
...
+    - grpcio >=1.8.6
     - mock >=2.0.0    # [py27]
-    - numpy >=1.12.1
+    - numpy >=1.13.3
     - protobuf >=3.4.0
     - six >=1.10.0
-    - tensorboard >=1.5.0,<1.6.0
+    - tensorboard >=1.6.0,<1.7.0
+    - termcolor >=1.1.0
```
Comment (jakirkham) [inline recipe/meta.yaml]: Strangely even the Windows wheels require `termcolor`. Though they don't require `colorama` on Windows, which is a bit unexpected. Not totally sure how this is suppose to work correctly on Windows.

**Comment by jakirkham** [comment]
@conda-forge-admin, please rerender.

**Comment by jakirkham** [comment]
@conda-forge-admin, please rerender.

**Comment by jakirkham** [comment]
@conda-forge-admin, please rerender.

**Comment by jakirkham** [comment]
@conda-forge-admin, please rerender.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -23,12 +23,16 @@ requirements:
...
     - backports.weakref >=1.0rc1
```
Comment (jjhelmus) [inline recipe/meta.yaml]: Although not mentioned in the setup.py file I believe `backports.weakref` is only required on Python <3.4.  That said I have no complaints about it being included as it doesn't hurt.

Comment (jakirkham) [reply recipe/meta.yaml]: Seems reasonable. Should be fixed. Thanks.

**Comment by jjhelmus** [comment]
LGTM, thanks for the update @jakirkham.

**Pull request merged**